android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java

```diff
@@ -1211,6 +1232,26 @@ public void setEnableInAppNotificationSound(boolean shouldPlaySound) {
         }
     }
 
+    private InstabugVideoRecordingButtonCorner getVideoRecordingButtonCorner(String cornerValue) {
+        InstabugVideoRecordingButtonCorner corner = InstabugVideoRecordingButtonCorner.BOTTOM_RIGHT;
+        try {
+            if (cornerValue.equals(BOTTOM_RIGHT)) {
+                corner = InstabugVideoRecordingButtonCorner.BOTTOM_RIGHT;
+            } else if (cornerValue.equals(BOTTOM_LEFT)) {
+                corner = InstabugVideoRecordingButtonCorner.BOTTOM_LEFT;
+            } else if (cornerValue.equals(TOP_LEFT)) {
+                corner = InstabugVideoRecordingButtonCorner.TOP_LEFT;
+            } else if (cornerValue.equals(TOP_RIGHT)) {
+                corner = InstabugVideoRecordingButtonCorner.TOP_RIGHT;
+            }
+
+            return corner;
+        } catch (Exception e) {
+            e.printStackTrace();
+            return corner;
```

Comment:
@salmatarzi it would be more elegant if you return the corner only once after the catch block (at the end of the method)